stdlib/public/SDK/Foundation/URLComponents.swift

```diff
@@ -291,6 +291,15 @@ public struct URLComponents : ReferenceConvertible, Hashable, Equatable, _Mutabl
         set { _applyMutation { $0.queryItems = newValue } }
     }
     
+	/// Returns an array of query items for this `URLComponents`, in the order in which they appear in the original query string. Any percent-encoding in a query item name or value is retained
```

Comment:
Nit: Please match existing formatting conventions for this file.

Reply:
Thanks. I've changed it.